db/seeds.rb

```diff
@@ -19,60 +19,131 @@
+trip5 = Trip.new(destination_id: 4,activity_id: 4,name: "Climbing in Pakistan",
+                 description: "Summit some of the most beautiful and untouched mountains on Earth.",
+                 trip_cost: 7850.00)
+trip5.image = image5
+trip5.save
+
+image6 = File.open('app/assets/images/argentina.jpg')
+trip6 = Trip.new(destination_id: 2,activity_id: 1,name: "Patagonia",
+                 description: "Travel to the end of the Earth and experience the strange beauty of Patagonia.",
+                 trip_cost: 36250.00)
+trip6.image = image6
+trip6.save
+
+image7 = File.open('app/assets/images/river.jpg')
+trip7 = Trip.new(destination_id: 1,activity_id: 2,name: "The Colorado Rapids",
+                 description: "Check an item off your bucket list with this once in a lifetime trip down the Colorado River.",
+                 trip_cost: 9550.00)
+trip7.image = image7
+trip7.save
+
+image8 = File.open('app/assets/images/basez.jpg')
+trip8 = Trip.new(destination_id: 6,activity_id: 5,name: "Base Jumping the Fjords",
+                 description: "Performing the most daring of stunts in the most isolated of places.",
+                 trip_cost: 38900.00)
+trip8.image = image8
+trip8.save
+
+image9 = File.open('app/assets/images/dolomites.jpg')
+trip9 = Trip.new(destination_id: 3,activity_id: 1,name: "Huts in the Dolomites",
+                 description: "Experience the old world culture of Northern Italy.",
+                 trip_cost: 9400.00)
+trip9.image = image9
+trip9.save
+
+image10 = File.open('app/assets/images/yosemite.jpg')
+trip10 = Trip.new(destination_id: 1,activity_id: 4,name: "Yosemite Granite",
+                 description: "Experience the world's natural playground and climb the classic Yosemite granite domes.",
+                 trip_cost: 21500.00)
+trip10.image = image10
+trip10.save
+
+image11 = File.open('app/assets/images/atlas.jpg')
+trip11 = Trip.new(destination_id: 5,activity_id: 4,name: "Climbing the High Atlas",
+                  description: "Spend your time in the thin air of the High Atlas Mountains in Northern Morocco.",
+                  trip_cost: 17900.00)
+trip11.image = image11
+trip11.save
+
+image12 = File.open('app/assets/images/montana.jpg')
+trip12 = Trip.new(destination_id: 1,activity_id: 6,name: "Hunting in Montana",
+                  description: "Experience one of the most vibrant displays of wildlife in the continental United States.",
+                  trip_cost: 33050.00)
+trip12.image = image12
+trip12.save
+
+image13 = File.open('app/assets/images/navua.jpg')
+trip13 = Trip.new(destination_id: 6,activity_id: 2,name: "The Navua River",
+                  description: "What's that? You didn't know that the tiny island of Fiji had such awesome river rafting? Well it does! Ride these rapids 65 kilometers to the south coast.",
+                  trip_cost: 25850.00)
+trip13.image = image13
+trip13.save
+
+image14 = File.open('app/assets/images/positano.jpg')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.